docs/en/week12/12-2.md

```diff
@@ -0,0 +1,184 @@
+</center>
+
+Taking a deeper look at the type of data we are presented with in machine translation, we can better understand how these applications can be mapped to machine learning techniques. For example, if we have a parallel dataset in the space of machine translation, then we have its equivalen in supervised learning. Second, we could have monolingual data in machine translation--this translates to semi-supervised learning within the machine elarning framework. Third, if we have multiple language paris in machine translation, we can closely compare this to multi-task learning within the ML space. And lastly, if we have multiple domains in machine translation, we can compare this to domain adaptation in machine learning. When you have many domains, you naturally also want to use different domain adaptation techniques
+
+
+## Case Studies
+**Translating from English to Nepali:** Let's start with a simple case study within the realm of supervsied learning. For example, let's say that we have a sentence in English and we want to trasnslate it to Nepali. This is similar to multitask learning in the sense that you have one task, and then you add another task you're interested in. Since we have multiple domains we can start thinking about domain adaptation techniques and analyzing which domain adaptation techniques are applicable to machine translation.
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-2/figure2.png" style="background-color:#DCDCDC;" /><br>
+</center>
+
+Beginning with a supervised learning setting, we can use the aforementioned example of translation. Our dataset is
+$$D = \{(x,y)_i\}_{i=1, ..., N}$$
+
+We can train this model using maximum likelihood, where we want to maximize $\{ y\vert x \} $.
+
+One way to represent this methodology is by the diagram, where you have a blue encoder that processes English sentences, and a red decoder that processes Nepali. From here, we want to compute our cross-entropy loss and update our model parameters. We may also want to regularize the model using either dropout or label smoothing. Additionally, we may need to regularize using the log loss
+
+$$-lo gp(y|x; \theta )$$
+
+*Note: Transliteration is not word per word translation. It means that you're using the charaters from one language to make a logical translation of the word in a different language.*
+
+**How can we improve generalization?**
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-2/figure3.png" style="background-color:#DCDCDC;" /><br>
+</center>
+
+One way to leverage an additional dataset is by trying to model $p(x)$ with a semi-supervised approach. One way to model $$p(x)$$ is via the denoising of an auto-encoder. This is particualrly useful because the encoder and decoder share a simlar machine translation methodology.
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-2/figure4.png" style="background-color:#DCDCDC;" /><br>
```

Comment:
Remove the math from the screenshot and type it out.
Figures are meant to be used for diagrams.
We have LaTeX for the math.
Here and elsewhere.